### ISSUE TITLE
Allowing a "Publish and Save" Option

### ISSUE THREAD

**Issue by randomtruffles** [netlify/netlify-cms#4054]
**Is your feature request related to a problem? Please describe.**

When working with multiple people, it's cool how editorial workflow allows saving drafts so others can view. However, it gets very "click-y" when you know you want to publish this version but have to press:

1. Save
2. Set Status
3. Ready
4. Publish
5. Publish Now
6. "Are You Sure You Want to Publish?" -> Yes

**Describe the solution you'd like**

If there was one single button to click to allow "Publish and Save" that would be great. 

**Describe alternatives you've considered**

Getting rid of Editorial Workflow would be a shame, because it allows saving work (ie. articles). But it is a lot of clicks.

**Additional context**

Thanks!

**Comment by erezrokah** [comment]
Hi @randomtruffles, do you mean "Save and Publish"? (save then publish)

**Comment by randomtruffles** [comment]
Yes, or something similar in the non-editorial mode where you can directly publish without doing all those steps (saving, draft status changing, publishing), especially for smaller changes.
